src/views/Explore.vue

```diff
@@ -113,9 +109,20 @@ export default {
     }
   },
   methods: {
-    loadMore() {
-      this.limit += 8;
+    scroll() {
+      window.onscroll = () => {
```

Comment:
You may want to create a function that we can reuse in utils to avoid duplicate code (i see the same method in Home.vue)

Reply:
Yes good point, I fixed that now.